docs/content/FormControl.mdx

```diff
@@ -32,6 +32,88 @@ import {MarkGithubIcon} from '@primer/octicons-react'
 </FormControl>
 ```
 
+### With a custom input
+
+<Note variant="warning">
+
+When rendering an input other than a form component from Primer, you must manually pass the attributes that make the form control accessible:
+
+- The input should have an ID
+- `FormControl.Label` should be associated with the text input by using `htmlFor`
+- If there is a caption, the input should be associated with the caption by passing the message's ID to `aria-describedby`
+- If there is a validation message, the input should be associated with the message by passing the message's ID to `aria-describedby`
+- If there is both a caption and a validation message, the input should be associated with the message by passing the both the validation message's ID and the caption's ID to `aria-describedby`
+- If the input's value is invalid, `aria-invalid={true}` should be passed to the input.
+- If the input is disabled, `disabled={true}` should be passed as true
+- If the input is required, `required={true}` should be passed as true
```

Comment:
copy is a bit redundant? 

Also, recommended `<input required />` instead of `<input required={true}/>` because they are equivalent

```suggestion
- If the input is disabled, `disabled` should be passed.
- If the input is required, `required` should be passed.
```

Reply:
Mistake on my part. I added the `={true}` and forgot to remove the last part of the sentence. I'm fine with just saying "If the input is disabled, `disabled` should be passed."